src/main/java/com/ibm/watson/modelmesh/Metrics.java

```diff
@@ -170,7 +170,7 @@ public PrometheusMetrics(Map<String, String> params) throws Exception {
                     }
                     break;
                 case "fq_names":
-                    shortNames = "true".equalsIgnoreCase(ent.getValue());
+                    shortNames = !"false".equalsIgnoreCase(ent.getValue());
```

Comment:
Hi @njhill -- sorry for the late review. But could there be a logic error here still? Double negations always trip me up a bit, but consider this example:

```Java
    String[] fq_names_values = {"true", "True", "TRUE", "false", "False", "FALSE", "", "off", "on", "N/A"};
    for (String fq_names_value: fq_names_values) {
        boolean shortNames = true;  // reset the default
        shortNames = !"false".equalsIgnoreCase(fq_names_value);
        System.out.println("fq_names='%s' => shortNames=%s".formatted(fq_names_value, shortNames));
    }
```

**Output:**
```
fq_names='true' => shortNames=true
fq_names='True' => shortNames=true
fq_names='TRUE' => shortNames=true
fq_names='false' => shortNames=false
fq_names='False' => shortNames=false
fq_names='FALSE' => shortNames=false
fq_names='' => shortNames=true
fq_names='off' => shortNames=true
fq_names='on' => shortNames=true
fq_names='N/A' => shortNames=true

```

Reply:
Thanks @ckadner, good catch. You are right, it should be `!"true` :-/. Feel free to open a PR to fix this or otherwise I will tomorrow...

Reply:
Okay. I create PR #77 to flip the boolean expression